features/messages/impl/src/main/res/values-de/translations.xml

```diff
@@ -4,6 +4,10 @@
     <item quantity="one">"%1$d Raumänderung"</item>
     <item quantity="other">"%1$d Raumänderungen"</item>
   </plurals>
+  <plurals name="screen_room_timeline_more_reactions">
+    <item quantity="one"></item>
```

Comment:
I don't understand why this empty case has been imported, there are other plurals with only the `other` case, and I don't see any difference in Localazy, wdyt @bmarty?

Reply:
I [asked for people](https://matrix.to/#/!kCCQTCfnABLKGGvQjo:matrix.org/$UUl0w-4Qi5KNKSi8prIZUOOZilmouDeM3lQ7sML4srs?via=matrix.org&via=element.io&via=lant.uk) to complete the translation, but it's not done yet.